### PR TITLE
chore: ng-add support

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: projects/sbb-esta/angular-public/schematics/collection.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "../../../../node_modules/@angular-devkit/schematics/collection-schema.json",
+  "schematics": {
+    "ng-add": {
+      "description": "add @sbb-esta/angular-public to project",
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/schematics/collection.json:5]: Start with Upper case

File: projects/sbb-esta/angular-public/schematics/ng-add/index.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export function ngAdd(options: Schema): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    if (getPackageJsonDependency(host, '@sbb-esta/angular-business')) {
+      context.logger.info(
+        '@sbb-esta/angular-business is already installed, please set up @sbb-esta/angular-public manually'
+      );
+      return;
+    }
+
+    // Since the Angular Material schematics depend on the schematic utility functions from the
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/schematics/ng-add/index.ts:18]: We are not Material 😄

File: projects/sbb-esta/angular-public/schematics/ng-add/setup-project.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  if (
+    !projectName ||
+    !workspace ||
+    !workspace.projects ||
+    !workspace.projects[projectName] ||
+    !workspace.projects[projectName].architect ||
+    !workspace.projects[projectName].architect![buildOrTest] ||
+    !workspace.projects[projectName].architect![buildOrTest]!.options
+  ) {
+    context.logger.error('Cannot add typography. Please add it manually to angular.json');
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/schematics/ng-add/setup-project.ts:51]: I'd prefer to state `Unable to add the typography. Please add an import to ${TYPOGRAPHY_CSS_PATH} to your project.`

File: projects/sbb-esta/angular-public/schematics/ng-add/setup-project.ts [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    return tree;
+  }
+
+  if (!(workspace.projects[projectName].architect![buildOrTest]!.options! as any).styles) {
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/schematics/ng-add/setup-project.ts:55]: Maybe assign options to a variable, so you don't have to use the full path every time

File: projects/sbb-esta/angular-business/schematics/rollup.config.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import { existsSync, readdirSync } from 'fs';
 import { join } from 'path';
 import copy from 'rollup-plugin-copy';
 
-const dist = join(__dirname, '../../../../dist/sbb-esta/angular-public/schematics');
+const dist = join(__dirname, '../../../../dist/sbb-esta/angular-business/schematics');
```
Comment (jeripeierSBB) [inline projects/sbb-esta/angular-business/schematics/rollup.config.js:6]: @kyubisation this was likely a bug in the current version, so no schematics were in angular-business

Comment (kyubisation) [reply projects/sbb-esta/angular-business/schematics/rollup.config.js:6]: Oops. Might be worth to release a patch release just for that...

**Pull request merged**